src/main/kotlin/org/rust/ide/spellchecker/StringLiteralTokenizer.kt

```diff
@@ -22,9 +22,8 @@ class StringLiteralTokenizer : EscapeSequenceTokenizer<LeafPsiElement>() {
         fun processTextWithEscapeSequences(element: LeafPsiElement, text: String, consumer: TokenConsumer) {
             val unescapedText = StringBuilder()
             val offsets = IntArray(text.length + 1)
-            CodeInsightUtilCore.parseStringCharacters(text, unescapedText, offsets)
-
-            processTextWithOffsets(element, consumer, unescapedText, offsets, 1)
```

Comment:
I'm not sure to be honest. It seems that in my screenshots in https://github.com/intellij-rust/intellij-rust/pull/79 the underlining was correct so I wonder if anything else changed. I guess some tests for this would be good.